test/smoke.test.ts

```diff
@@ -209,52 +265,46 @@ describe("Smoke test", () => {
         title: "My test issue",
       });
 
-      expect(HttpsProxyAgent).toHaveBeenCalled();
-
       const [call] = fetchSandbox.calls();
       expect(call[0]).toEqual(
         "https://api.github.com/repos/octocat/hello-world/issues",
       );
-      expect((call[1] as RequestOptions).agent).toBeInstanceOf(HttpsProxyAgent);
     },
   );
+  describe("customFetch", () => {
+    afterAll(() => {
+      delete process.env.HTTPS_PROXY;
+      jest.clearAllMocks();
+    });
 
-  it("Uses the explicitly provided request.agent value if it's provided", async () => {
```

Comment:
I've removed this test as it wasn't functional in its current state and the intent is tested by "should call undiciFetch with the correct dispatcher".